levels/sre.tex

```diff
@@ -0,0 +1,368 @@
+\documentclass{article}
+
+\usepackage[table]{xcolor}
+\usepackage[paperwidth=800mm, paperheight=575mm]{geometry}
+
+% Don't show page numbers.
+\pagenumbering{gobble}
+
+% Column settings for skills table.
+\newcolumntype{C}{p{20em}}
+
+\begin{document}
+
+% Title of matrix.
+\begin{center}
+\begin{huge}
+Site Reliability Engineering (SRE)
+\end{huge}
+\end{center}
+
+\bigbreak
+
+% Alternating row colors.
+\rowcolors{2}{gray!80!gray!50}{gray!50!gray!20}
+
+% Padding for each cell.
+{\renewcommand{\arraystretch}{2}
+
+% Skills table.
+\begin{center}
+\begin{tabular}{|C|C|C|C|C|C|C|C|}
+\hline
+\rowcolor{blue!60!black!10}
+    \textbf{Level/Skill}
+    &
+    \cellcolor{blue!25}
+    \textbf{Product Output}
+    &
+    \textbf{Communication/Writing}
+    &
+    \textbf{Networking}
+    &
+    \textbf{Security}
+    &
+    \textbf{Systems Engineering}
+    &
+    \textbf{Site Reliability Engineering (SRE)}
+    \\
+\hline
+    % Level
+    1
+    &
+
+    % Product Output
+    \cellcolor{blue!25}
+    Creates a design document based on well-defined scoped requirements and
+    implements it.
+    &
+
+    % Communication
+    Reports progress on a regular basis as required by the team's operational
+    requirements. Actively solicits feedback.Participates on interview panels.
```

Comment:
We don't have interview panels in the systems levels - don't think we need it here either - although I don't feel strongly about it either way.